invenio_communities/utils.py

```diff
@@ -109,3 +112,15 @@ def on_datastore_post_commit(sender, session):
             users = role.users.all()
             for user in users:
                 on_user_membership_change(Identity(user.id))
+
+
+def humanize_byte_size(size):
+    """Converts bytes to largest unit (e.g., KB, MB, GB)."""
+    s = Decimal(size)
+    for unit in ["B", "KB", "MB", "GB", "TB", "PB"]:
+        if s < BYTES_PER_UNIT:
+            with localcontext() as ctx:
+                ctx.rounding = ROUND_HALF_UP
+                s = s.quantize(Decimal("0.00"))
```

Comment:
Does it make sense to have all calls to Decimal() use a string argument? It seems to be the best practice is to use a string, but from my perspective it doesn't make a difference for whole number values like 1000. I guess consistency would be a reason.

Reply:
For small d decimal, a string is the way to go, but for whole numbers it doesn't matter too much like you are saying. It's not one of those consistency things that bothers me beyond that.